ios/ZulipMobile/ZLPNotificationsBridge.m

```diff
@@ -1,13 +1,17 @@
 #import "React/RCTBridgeModule.h"
+#import "React/RCTEventEmitter.h"
 
-// Register the ZLPNotifications implementation with React Native, needed
-// because ZLPNotifications is in Swift:
+// Register our Swift modules with React Native:
 //   https://reactnative.dev/docs/0.68/native-modules-ios#exporting-swift
-@interface RCT_EXTERN_MODULE(ZLPNotifications, NSObject)
+
+@interface RCT_EXTERN_MODULE(ZLPNotificationsStatus, NSObject)
 
     RCT_EXTERN_METHOD(areNotificationsAuthorized:
         (RCTPromiseResolveBlock) resolve
         rejecter: (RCTPromiseRejectBlock) reject
     )
 
 @end
+
+@interface RCT_EXTERN_MODULE(ZLPNotificationsEvents, RCTEventEmitter)
+@end
```

Comment:
nit: put these in the same order as the classes appear in the Swift file

(I don't have a preference which order that is.)